src/components/text-input/_text-input.scss

```diff
@@ -208,6 +208,7 @@
     outline: none;
     background-color: $disabled-01;
     border-bottom: 1px solid transparent;
+    // color: $disabled-02;
```

Comment:
Did you mean to uncomment this @emyarod?

Reply:
yes, should be fixed now